typings/index.d.ts

```diff
@@ -1605,6 +1604,7 @@ declare module 'discord.js' {
     public voiceManager: ClientVoiceManager;
     public disconnect(): void;
     public play(input: VoiceBroadcast | Readable | string, options?: StreamOptions): StreamDispatcher;
+    public setSpeaking(value: BitFieldResolvable<SpeakingString>): void;
```

Comment:
Can you move this back with the other `set` methods?

Reply:
I feel like having private and public function declarations combined with one another is messy